slices/openjdk-8-jre-headless.yaml

```diff
@@ -27,8 +27,10 @@ slices:
       /usr/lib/jvm/java-8-openjdk-*/jre/lib/*/libverify.so:
       /usr/lib/jvm/java-8-openjdk-*/jre/lib/*/libzip.so:
       /usr/lib/jvm/java-8-openjdk-*/jre/lib/*/libjsig.so:
-      /usr/lib/jvm/java-8-openjdk-*/jre/lib/*/server/libjsig.so:
-      /usr/lib/jvm/java-8-openjdk-*/jre/lib/*/server/libjvm.so:
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/*/client/libjsig.so: {arch: armhf}
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/*/client/libjvm.so: {arch: armhf}
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/*/server/libjsig.so: {arch: [amd64,arm64,ppc64el]}
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/*/server/libjvm.so: {arch: [amd64,arm64,ppc64el]}
```

Comment:
nit: what do you think about unfolding the values and adding `arch` in a below line? The lines **might** be too long now for future lint jobs.

Reply:
sounds good -> https://github.com/canonical/chisel-releases/pull/149/commits/7641e46287745b642efdc1f00c269c0260b15a56